renderdoc/core/replay_proxy.h

```diff
@@ -504,7 +505,17 @@ class ReplayProxy : public IReplayDriver, Callstack::StackResolver
   };
   set<TextureCacheEntry> m_TextureProxyCache;
   set<ResourceId> m_LocalTextures;
-  map<ResourceId, ResourceId> m_ProxyTextureIds;
+
+  struct ProxyId_GetTextureDataParams : GetTextureDataParams
```

Comment:
I'd prefer a more descriptive name such as `ProxyTextureProperties` or `ProxyTextureParams` here rather than naming explicitly after the pair of properties that we have currently which is more likely to get out of date.

The map could then also just be `m_ProxyTextures` instead of `m_ProxyTextureIds`.

Also, please add the GetTextureDataParams as a member next to the id, rather than inheriting from it.